src/SFA.DAS.AssessorService.Database/Tables/Privileges.sql

```diff
@@ -1,7 +1,8 @@
 ﻿CREATE TABLE [dbo].[Privileges](
 	[Id] [uniqueidentifier] NOT NULL,
```

Comment:
Could we make Id have a Default of NEWID() ?

Reply:
Possibly in the future, but for now these Privileges aren't created anywhere in code and are just 'hard coded' into the db. 👍